bw2io/importers/simapro_lcia_csv.py

```diff
@@ -9,8 +9,37 @@
 )
 from .base_lcia import LCIAImporter
 
-
 class SimaProLCIACSVImporter(LCIAImporter):
+    """Importer for SimaPro CSV LCIA data format.
+
+    Parameters
+    ----------
+    filepath : str
+        Path to the SimaPro CSV LCIA file.
+    biosphere : str, optional
+        Name of the biosphere database to use. Default is None, which uses the current
+        project's default biosphere.
+    delimiter : str, optional
+        Delimiter used in the CSV file. Default is ';'.
+    encoding : str, optional
+        Character encoding used in the CSV file. Default is 'latin-1'.
+    normalize_biosphere : bool, optional
+        Whether to normalize biosphere flows using the included strategies.
+        Default is True.
+
+    Notes
+    -----
+    This importer extracts SimaPro CSV LCIA data using the
```

Comment:
Redundant? Better to make the short description above more precise.

Reply:
Fixed with https://github.com/brightway-lca/brightway2-io/pull/166/commits/136c40ca7ca064fccadfcb2d50b44788d27ec505